drivers/net/ethernet/intel/idpf/idpf_eth_common.h

```diff
@@ -0,0 +1,254 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/* Copyright (C) 2023 Intel Corporation */
+
+#ifndef _IDPF_ETH_COMMON_H_
+#define _IDPF_ETH_COMMON_H_
+
+#define IDPF_LARGE_MAX_Q			256
+/* Mailbox Queue */
+#define IDPF_MAX_MBXQ				1
+
+#define GETMAXVAL(num_bits)		GENMASK((num_bits) - 1, 0)
+#define IDPF_MAX_BUFQS_PER_RXQ_GRP		2
+
+#define IDPF_TX_MIN_PKT_LEN		17
+#define IDPF_NO_FREE_SLOT		0xffff
+#define IDPF_NUM_CHUNKS_PER_MSG(struct_sz, chunk_sz)	\
+	((IDPF_CTLQ_MAX_BUF_LEN - (struct_sz)) / (chunk_sz))
+
+#define IDPF_MAX_Q				16
+#define IDPF_MIN_Q				2
+
+/**
+ * enum idpf_vport_type
+ * @IDPF_DEFAULT_VPORT: Default vport
+ * @IDPF_DYNAMIC_VPORT: Dynamic vport
+ */
+enum idpf_vport_type {
+	IDPF_DEFAULT_VPORT = 1,
+	IDPF_DYNAMIC_VPORT
+};
+
+/**
+ * enum idpf_queue_type
+ * @IDPF_QUEUE_DEFAULT_VPORT: Queue for Default vport
+ * @IDPF_QUEUE_DYNAMIC_VPORT: Queue for Dynamic vport
+ */
+enum idpf_queue_type {
+	IDPF_QUEUE_DEFAULT_VPORT = 1,
+	IDPF_QUEUE_DYNAMIC_VPORT
+};
+
+/**
+ * enum idpf_eth_idc_event_type
+ * Event type IDPF_ETH_IDC_EVENT_ALL_VPORTS are routed to all ports, while
+ * the event type IDPF_ETH_IDC_EVENT_SINGLE_VPORT is routed to only the
+ * intended port.
+ */
+enum idpf_eth_idc_event_type {
```

Comment:
This enum is used only by pci driver. So cannot be part of common.h. Move it to idpf.h

Reply:
Moved.